approval_frame/settings.py

```diff
@@ -9,7 +9,10 @@
 SECRET_KEY = 'jndet@)ed9ge4+!k605hc+$0tjb8u5oe)pdjab)-b3lsot_xnf'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+# For development purposes, set the DEBUG_* variables to True in a 
+# local_settings.py file, which we .gitignore. This file should be added
+# in the same directory that contains this settings.py file.
+DEBUG = False
 TEMPLATE_DEBUG = DEBUG
```

Comment:
That sounds good. Can you also add in the README.md that if you want Django to run on a public IP then you would have to update the allowed hosts array?
Thanks a lot! That would be the last change.